app/assets/javascripts/discourse/app/components/modal/topic-bulk-actions.js

```diff
@@ -196,6 +196,16 @@ export default class TopicBulkActions extends Component {
     },
   ];
 
+  constructor() {
+    super(...arguments);
+
+    if (this.args.model.initialAction) {
```

Comment:
I think this check is redundant since we are checking `initialAction` against some value.